### ISSUE TITLE
Make sure the largest stable version number is tagged as the latest dist-tag

### ISSUE THREAD

**Issue by JakeChampion** [Financial-Times/origami-ci-tools#10]
## What
Currently the most recently tagged version is the one which gets tagged "latest" in npm, which means that version is the one which is downloaded if someone does `npm install thing` without specifying a version.

To work around this issue, when working on older versions of a package and publishing a minor/patch update to the version, the publisher has to take an extra step after publishing to make sure the largest stable version gets retagged as the "latest" version.

We should automate that step inside origami-ci-tools somehow :-)

**Comment by JakeChampion** [comment]
I think we can solve this using this:
``` bash
$ npm info . versions --json
> [
  "1.0.0",
  "1.0.1",
  "1.0.2-peers.0",
  "1.0.2-peers.1",
  "1.0.2-peers.2",
  "1.0.2-peers.3",
  "1.0.2-peers.4",
  "1.0.2-peers.5",
  "1.1.0",
  "1.1.1"
]
```

**Comment by JakeChampion** [comment]
Fixed via #15
